learing_journal/tests.py

```diff
@@ -103,21 +104,34 @@ def test_lists_body(new_session):
 def test_lists_date(new_session):
     '''tests wether list() pull out correct data from db'''
     from .views.default import home
-    new_session.add(MyModel(title='test1', body='test2', date='test3'))
+    new_session.add(Journal(title='test1', body='test2',
+                    date=datetime.datetime.strptime('August 23, 2016', '%B %d, %Y')))
     new_session.flush()
     result = home(dummy_http_request(new_session))
     for entry in result['entries']:
-        assert entry.date == 'test3'
+        assert entry.date.strftime('%B %d, %Y') == 'August 23, 2016'
```

Comment:
This is a decent test, as you're making sure that your date string shows up formatted properly.